api/src/main/java/org/apache/cloudstack/api/response/RemoteDiagnosticsResponse.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * // Licensed to the Apache Software Foundation (ASF) under one
+ * // or more contributor license agreements.  See the NOTICE file
+ * // distributed with this work for additional information
+ * // regarding copyright ownership.  The ASF licenses this file
+ * // to you under the Apache License, Version 2.0 (the
+ * // "License"); you may not use this file except in compliance
+ * // with the License.  You may obtain a copy of the License at
+ * //
+ * //   http://www.apache.org/licenses/LICENSE-2.0
+ * //
+ * // Unless required by applicable law or agreed to in writing,
+ * // software distributed under the License is distributed on an
+ * // "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * // KIND, either express or implied.  See the License for the
+ * // specific language governing permissions and limitations
+ * // under the License.
+ */
+
+package org.apache.cloudstack.api.response;
+
+import com.cloud.serializer.Param;
+import com.cloud.vm.VirtualMachine;
+import com.google.gson.annotations.SerializedName;
+import org.apache.cloudstack.api.ApiConstants;
+import org.apache.cloudstack.api.BaseResponse;
+import org.apache.cloudstack.api.EntityReference;
+
+@EntityReference(value = VirtualMachine.class)
+public class RemoteDiagnosticsResponse extends BaseResponse {
+    @SerializedName(ApiConstants.DETAILS)
+    @Param(description = "Script execution result")
+    private String details;
+
+    @SerializedName(ApiConstants.RESULT)
+    @Param(description = "true if operation is executed successfully")
+    private Boolean success;
+
+    @SerializedName("command")
+    @Param(description = "Command passed to script")
```

Comment:
this description doesn't tell me what I expect to read here. Is it the exact RemoteDiagnosticsCmd.type? or the cmd line as eventually executed remotely, including the standard and extra parameters, or even path?

Reply:
I put this the for debug purposes, I will remove it in the final product. But this is the command line is eventually executed remotely. 

E.g.  command = ping 8.8.8.8 -c 5

Reply:
ok, that might be usefull for an administrator in more elaborate cases as well